assets/sass/05-blocks/utilities/_style.scss

```
@@ -50,23 +60,33 @@
  */
 .alignright {
 
-	/*rtl:ignore*/
-	float: right;
 	margin-top: 0;
 	margin-bottom: var(--global--spacing-vertical);
-
-	/*rtl:ignore*/
-	margin-left: var(--global--spacing-horizontal);
 }
 
 // Targeting the .entry-content class is necessary to ensure these styles
 // only apply when the block isn't nested.
 .entry-content > .alignright {
 	max-width: var(--responsive--aligndefault-width);
-	@extend %responsive-alignright-mobile;
 	@extend %responsive-alignright;
 }
 
+@include media(mobile) {
+
+	.alignright {
+
+		/*rtl:ignore*/
+		float: right;
+
+		/*rtl:ignore*/
+		margin-left: var(--global--spacing-horizontal);
+	}
+
+	.entry-content > .alignleft {
```

Is this meant to be `alignright?`

---

tested and confirmed that it only works if this class is replaced.
```suggestion
	.entry-content > .alignright {
```

---

@carolinan Oops! Thanks for catching, I thought I had done that :sweat_smile: